.retest/recheck.ignore.js

```diff
@@ -12,6 +12,9 @@ function contains(array, key) {
 }
 
 function matches(element, diff) {
+	if (diff == null) {
```

Comment:
Shouldn't we have a separate `match(element)` for this?

Also please use truthy checks if it is `undefined` since it is possible to call match(el) with this definition where JavaScript would insert `undefined` into missing parameters.
```suggestion
	if (!diff) {
```